GAE/src/org/waterforpeople/mapping/dao/SurveyInstanceDAO.java

```diff
@@ -629,13 +629,14 @@ public void deleteSurveyInstance(SurveyInstance surveyInstance) {
         QuestionAnswerStoreDao qasDao = new QuestionAnswerStoreDao();
         QuestionDao qDao = new QuestionDao();
         List<QuestionAnswerStore> qasList = qasDao.listBySurveyInstance(surveyInstanceId);
+        SurveyQuestionSummaryDao summDao = new SurveyQuestionSummaryDao();
```

Comment:
@mtwestra is this class declared in the `webapp-context.xml` ? is the way of declaring it